docs/src/dev/provider/index.asciidoc

```diff
@@ -1007,56 +1167,41 @@ dependent on the operation being performed.  A `RequestMessage` has these fields
 [width="100%",cols="3,10",options="header"]
 |=========================================================
 |Key |Description
-|requestId |A link:http://en.wikipedia.org/wiki/Globally_unique_identifier[UUID] representing the unique identification for the request.
-|op |The name of the "operation" to execute based on the available `OpProcessor` configured in the Gremlin Server.  To evaluate a script, use `eval`.
-|processor |The name of the `OpProcessor` to utilize. The default `OpProcessor` for evaluating scripts is unnamed and therefore script evaluation purposes, this value can be an empty string.
-|args |A `Map` of arbitrary parameters to pass to Gremlin Server.  The requirements for the contents of this `Map` are dependent on the `op` selected.
+|gremlin |The script to be executed by the server.
+|language |The script's language.  The default is `gremlin-lang`.
+|timeoutMs |The maximum time a query is allowed to execute in milliseconds.  The default value is server dependent.
+|materializeProperties |Determines if properties will be included in `Element` results.
+|bindings |A `Map` of arbitrary parameters to pass to the server.  The requirements for the contents of this `Map` are dependent on the `op` selected.
+|g |The name of the `GraphTraversalSource` to use on the server.  The default is `g`.
 |=========================================================
 
-This message can be serialized in any fashion that is supported by Gremlin Server.  New serialization methods can
-be plugged in by implementing a `ServiceLoader` enabled `MessageSerializer`, however Gremlin Server provides for
+This message can be serialized in any fashion that is supported by Gremlin Server.  Gremlin Server provides for
 JSON serialization by default which will be good enough for purposes of most developers building drivers.
 A `RequestMessage` to evaluate a script with variable bindings looks like this in JSON:
 
 [source,js]
 ----
-{ "requestId":"1d6d02bd-8e56-421d-9438-3bd6d0079ff1",
-  "op":"eval",
-  "processor":"",
-  "args":{"gremlin":"g.V(x).out()",
-          "bindings":{"x":1},
-          "language":"gremlin-groovy"}}
+{ "gremlin":"g.V(x).out()",
+  "bindings":{"x":1},
+  "language":"gremlin-groovy"}}
 ----
 
 The above JSON represents the "body" of the request to send to Gremlin Server. When sending this "body" over
```

Comment:
more on the point of JSON - you called it a pseudo-JSON earlier in your writing. perhaps it is a "pseudo-request" around which you write this section of doc?

Reply:
this section has been removed.